src/main/java/com/teradata/tpcds/Options.java

```diff
@@ -32,7 +32,7 @@
     public static final boolean DEFAULT_OVERWRITE = false;
 
     @Option(name = {"--scale", "-s"}, title = "scale", description = "Volume of data to generate in GB (Default: 1)")
-    public int scale = DEFAULT_SCALE;
+    public float scale = DEFAULT_SCALE;
```

Comment:
There's no super compelling reason for using float over double. The scales that we're going to use fit well within float's range (our max scale being 100 TB or 100 000 GB) and float has a smaller footprint since it is 32 bits. However, I realize that using float is a little awkward given that double is usually the default type used in Java for decimal values so I'd be happy to change it over to double.